plugin/editorconfig.py

```diff
@@ -8,8 +8,25 @@
         vim.command('let l:ret = 2')
         raise
 
+    # Converts the path to windows when we are in mingw or cygwin
+    def maybe_to_win(filepath):
+        if int(vim.eval("has('win32unix') || has('win64unix')")) != 0:
+            if filepath[0] == '/':
+                # hacky way of making a unix path out of a linux path
+                filepath = filepath[1] + ':' + filepath[2:]
+
+        return filepath
+
+
     try:
-        sys.path.insert(0, vim.eval('a:editorconfig_core_py_dir'))
+        verbose = int(vim.eval('g:EditorConfig_verbose')) != 0
```

Comment:
Adding a global variable with such a simple name can easily cause conflicts with other extensions -- Vim uses a single namespace for all global Python variables.

Reply:
You can use `ec_data['verbose']` instead.

Reply:
I did not know they shared a common namespace. I'll improve it.